README.md

```diff
@@ -1,3 +1,18 @@
-# ovsdb-etcd
+[![Build Status](https://travis-ci.com/IBM/ovsdb-etcd.svg?branch=master)](https://travis-ci.com/IBM/ovsdb-etcd "Travis")
+[![Go Report Card](https://goreportcard.com/badge/github.com/IBM/ovsdb-etcd)](https://goreportcard.com/report/github.com/IBM/ovsdb-etcd)
 
-A PoC project to store OVN database date into an etcd cluster   
\ No newline at end of file
+# OVSDB ETCD Backend (PoC)
+
+A PoC project to store OVN database date into an etcd cluster, as by doing so
+we can levarge the advantages of the widly used ETCD key-value store. As an
```

Comment:
we can levarge the advantages of the widly used ETCD key-value store ->
we can leverage the advantages of the widely used distributed key-value store - etcd.

Reply:
ok